_maps/RandomRuins/SpaceRuins/Fast_Food.dmm

```diff
@@ -598,7 +587,10 @@
 "ck" = (
 /obj/machinery/light/floor,
 /obj/effect/decal/cleanable/dirt/dust,
-/turf/open/water,
+/turf/open/water{
+	initial_gas_mix = "o2=22;n2=82;TEMP=293.15";
+	planetary_atmos = 0
+	},
```

Comment:
Making a subtype of `turf/open/water` with these attributes is a standard way to solve this problem. Append the resulting subtype to code/game/turfs/open/water.dm

Reply:
making a whole new subtype just to get it used on one ruin? ok

Reply:
Most people don't realize, but varedit is ~~technically~~ a sin.